controllers/patient.controller.js

```diff
@@ -21,4 +31,25 @@ const post = async (req, res) => {
   }
 };
 
-module.exports = { post, get };
+const put = async (req, res) => {
+  try {
+    // const value = await patientSchema.validateAsync(req.body);
+    const patient = await patientServices.editPatient(req.body);
+    return res.status(201).json({ id: patient.id });
+  } catch (err) {
+    return resolveError(err, res);
+  }
+};
+
+const deletePatient = async (req, res) => {
+  try {
+    const num = await patientServices.deletePatient(req.params);
+    return res.status(200).json({ num });
+  } catch (err) {
+    return resolveError(err, res);
+  }
+};
+
+module.exports = {
+  post, get, getPatient, deletePatient, put
```

Comment:
naming here is bad. let's rename this:
- `post` -> `create`
- `get` -> `list`
- `getPatient` -> `retrieve`
- `deletePatient` -> `delete`
- `put` -> `update`